checkout-block/class-blocks-integration.php

```diff
@@ -69,7 +69,7 @@ public function register_block_frontend_scripts() {
 
 		wp_register_script(
 			'checkout-block-frontend',
-			trailingslashit( plugin_dir_url( __DIR__ ) ) . 'includes/js/checkout-block-frontend.js',
+			trailingslashit( plugin_dir_url( __DIR__ ) ) . 'src/frontend.js',
```

Comment:
Add the build folder checkout-block-frontend.js file

Reply:
See this final commit : https://github.com/TycheSoftwares/checkout-fees-for-woocommerce/pull/202/files#diff-39d28b094398981714ade49c6aea7b4e0d26e68045cd7db181aea61a1a4f86d2R72